### PR TITLE
Update contribution, licensing and coypright guidelines

### PR DESCRIPTION
**Pull request by dirkmueller**
Suggest to keep main copyrights in the top level README.md, with the option to have file level copyrights where sections are selectively maintained and contributed to.

Clarify Code of Conduct and contribution references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +11,31 @@ While we recognize that the size of these directories might grow over time and f
 For more information about this project see the #documentation channel on the OpenELA Slack.
 
 ## Contributing
-Contributions are welcome. Please fork and submit a pull request for any changes. 
+
+This project is open for contributions and Contributions are very welcome.
```
Comment (SergeCroise) [inline README.md:15]:
```suggestion
This project is open for contributions and contributions are very welcome.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
 ## Contribute documentation
 
-This is a community project and we welcome community members to contribute 
-documentation to the project.
+This is a community project and we welcome community members to contribute
+documentation to the project. Contributions are done [by opening a pull request against the project](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+Any contribution requires a [Developer Certificate of Origin](https://developercertificate.org/) (a Signed-off-By tag in the commit message) and
+must adhere to the [project license](./LICENSE).
+
```
Comment (sspencerwire) [inline CONTRIBUTING.md:24]: Looks like lines 22 and 23 should be combined, as in there may  be a CR/LF after "and" on 22 that should be removed.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,34 +1,44 @@
...
-https://github.com/openela/governance/blob/main/contributors_guidelines.md
 
-## Did you find a bug?
+All contributions to this repository and to OpenELA in general follow
+[the OpenELA Code of Conduct](https://github.com/openela/governance/blob/main/code_of_conduct.md).
+
+
+## Did you find an issue?
+
+Issues, be it a defect, a missing documentation part or any enhancement request, please [open an issue on GitHub](https://github.com/openela/openela-documentation/issues). Please provide as much detail as you can
```
Comment (ol-roboro) [inline CONTRIBUTING.md:11]: Possibly an odd line break here, but probably not serious.

**Review by ol-roboro — COMMENTED** [review]
I am happy with the changes proposed here. There are a few odd line breaks, which you might want to fix before we merge, but otherwise go ahead with the merge.
I am currently away, but will update our sources for the agreed copyright changes when I return.

**Review by wsoyinka — APPROVED** [review]

**Review by wsoyinka — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (dirkmueller) [reply CONTRIBUTING.md:11]: oh, thanks for noticing. I've fixed this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (dirkmueller) [reply CONTRIBUTING.md:25]: actually multiline paragraphs is valid in markdown..

**Pull request merged**